share/functions/funced.fish

```diff
@@ -1,3 +1,16 @@
+function __funced_md5
+    if type -q md5sum
+        # GNU systems
+        md5sum $argv[1] | cut -d' ' -f1
```

Comment:
Good point but after reading some docs I realised that `string split` would be more readable :)